### DISCUSSION TITLE
Surveillance Station 9.x release

### DISCUSSION THREAD

**Discussion by Grandma-Betty** [djdd87/SynoAI#126] [General]
Synology released Synology Surveillance Station 9.x today.
Just wanted to let y'all know that SynoAI still works flawlessly in my case👍

**Comment by mkammes** [comment]
Is it still beta or is this a full release? @Grandma-Betty

**Reply by Grandma-Betty** [reply]
@mkammes Full release.

**Reply by mkammes** [reply]
I had to do some manual updates, as the update wasn't available automatically. It also required me to move from DSM 7.0.1-x to 7.1-x. You also need to install the Advanced Media Extension pack (to handle h.265 and AAC audio) and then log in to the Synology website on the DSM to get the codec pack (which still didn't work right, I had to do a manual install).

All of that being said, SynoAI is working fine here!
